internal/datastore/mysql/migrations/zz_migration.0004_indexed_gc.go

```diff
@@ -0,0 +1,16 @@
+package migrations
```

Comment:
Maybe squash these migrations since this is a new driver?

Reply:
addressed in https://github.com/authzed/spicedb/pull/525/commits/055f8094661c1b5bfd76b4c3e55e6e7f4840a4d0